### ISSUE TITLE
HTTPErrors component

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-cloud-cognitive#415]
# Name of component or pattern
HTTPError
https://pages.github.ibm.com/cdai-design/pal/patterns/http-errors/usage
## Brief description
`HTTP errors are used in an attempt to access something that isn’t available or the user doesn’t have permission for.`

This component will serve as an updated version to the one found [here](https://deploy-preview-412--ibm-cloud-cognitive.netlify.app/?path=/story/legacy-cd-ai-idehttperrors--default) which is only receiving legacy support from this point forward.

## Details

[Link to Design](https://pages.github.ibm.com/cdai-design/pal/patterns/http-errors/usage)
[Link to Legacy CD&AI component](https://deploy-preview-412--ibm-cloud-cognitive.netlify.app/?path=/story/legacy-cd-ai-idehttperrors--default)

### Links to other materials

e.g. sketch files, icons, images, prototypes

## Design Maintainer(s)

- Owner/main maintainer(s)
- Second/backup maintainer(s)

## Tasks

Before starting work on this epic, please review and complete the following.

- [x] Initial review of design/existing code

### Working in Cloud and Cognitive package

- [x] Have reviewed our
      [contribution guidelines](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/master/.github/CONTRIBUTING.md)?
- [x] Have you noted our
      [reviewer guidelines](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/master/.github/REVIEWERS_GUIDELINES.md)?
- [x] Did you use the CLI to scaffold your new component?

**Comment by gregorcowieibm** [comment]
Design have reviewed and it looks good to go.

**Comment by matthewgallo** [comment]
Component has been released. Anything that comes up can be addressed in a new issue.
